scripts/fast-reboot-dump.py

```diff
@@ -80,23 +80,45 @@ def get_bridge_port_id_2_port_id(db):
 
     return bridge_port_id_2_port_id
 
-def get_map_port_id_2_iface_name(db):
+def get_lag_name(member_name, app_db):
```

Comment:
I suggest the following function signature:
```suggestion
def get_lag_by_member(app_db, member_name):
```

Reply:
Fixed